devops/Makefile

```diff
@@ -43,6 +43,110 @@ define DB_NAME
 "SIMSDB"
 endef
 
+define FORMIO_FORMS_NAME
```

Comment:
Is there any reason to chose to "define" these variables instead of use the .env file alongside with the default values as other in the make file?
```
export APP_NAME := $(or $(PROJECT_NAME), sims)
export BUILD_NAMESPACE := $(or $(BUILD_NAMESPACE), 0c27fb-tools)
```

Reply:
We can use it, but the main reason I used define is that these names or paths are not going to change for different deployments.